File: packages/flutter/ios/Classes/VoicevoxFlutterPlugin.m
#import "VoicevoxFlutterPlugin.h"

@implementation VoicevoxFlutterPlugin

+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
  // FFI プラグインのためメソッドチャンネル登録は不要。
  // このクラスは登録機構(GeneratedPluginRegistrant)から呼ばれる
  // エントリポイントとしてのみ存在する。
}

@end

File: packages/flutter/macos/Classes/VoicevoxFlutterPlugin.swift
// FFI プラグインのため Dart 側から dart:ffi で直接ネイティブライブラリを呼ぶ。
// このファイルは CocoaPods プラグインの最小ソースとして存在する。
import FlutterMacOS
import Foundation

public class VoicevoxFlutterPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
    // FFI プラグインのためメソッドチャンネル登録は不要。
  }
}
